app/models/scorecard.py

```diff
@@ -1,12 +1,11 @@
+from app.database.base_class import Base
 from sqlalchemy import Column, Integer, String
 
-from ...database import Base
-
 
 class Scorecard(Base):
     id = Column(Integer, primary_key=True, index=True)
-    name = Column(String(50), nullable=False)
-    description = Column(String(200), nullable=False)
+    name = Column(String, nullable=False)
```

Comment:
Why did you remove the limit on the allowed number of characters for each column ?